helics/meta.yaml

```diff
@@ -1,30 +1,50 @@
 {% set name = "helics" %}
-{% set version = "2.6.1" %}
+{% set version = "2.6.1.post0.dev0" %}
 
 package:
   name: "{{ name|lower }}"
   version: "{{ version }}"
 
 build:
   number: 0
+  features:
+    - vc9  [win and py27]
+    - vc10 [win and py34]
+    - vc14 [win and py>=35]
+
+source:
+  - git_url: https://github.com/GMLC-TDC/HELICS-src
```

Comment:
```suggestion
  - git_url: https://github.com/GMLC-TDC/HELICS
```

Reply:
Resolved in https://github.com/GMLC-TDC/helics-conda/pull/9/commits/08c4638e624023a5b9434694cabd112015d655da.